src/functionalTest/resources/application.yaml

```diff
@@ -0,0 +1,53 @@
+management:
+  endpoint:
+    health:
+      show-details: "always"
+  endpoints:
+    web:
+      base-path: "/"
+      exposure:
+        include: health, info, prometheus
+  security:
+    enabled: false
+
+spring:
+  application:
+    name: Bulk Scan Payment Processor
+  main:
+    # without this, Spring can't properly resolve feign clients when IDAM and CCD clients are both included
+    # TODO: remove when possible
+    allow-bean-definition-overriding: true
+
+azure:
+  servicebus:
+    payments:
+      connection-string: ${PAYMENTS_QUEUE_READ_CONNECTION_STRING}
+      write-connection-string: ${PAYMENTS_QUEUE_WRITE_CONNECTION_STRING}
+      queue-name: payments
+      max-delivery-count: 5
```

Comment:
I guess `connection-string` and `max-delivery-count` should never be referenced by functional tests as application and we can safely remove them.